### ISSUE TITLE
Actively deprecating more 'openssl' sub-commands

### ISSUE THREAD

**Issue by levitte** [openssl/openssl#7321]
Quite a number of commands are documented as superseded by others in `doc/man1/openssl.pod`, but some are still not.  For example, `openssl rsa` is superseded by `openssl pkey`.

Things that I believe we need to do:

- complete the documentation in this regard, i.e. add information on what supersedes what where this information is currently missing.
- change those programs to display a deprecation warning, and what command to use instead.
- plan for removal in some future major OpenSSL release.

**Comment by levitte** [comment]
The main reason to want to do this is that the old commands are a maintenance burden, or a source off bit rot.  Either is bad.

**Comment by nhorman** [comment]
Marking as inactive, to be closed at the end of 3.4 dev, barring further input

**Comment by t8m** [comment]
I do not think we are going to do this any time soon. Besides this is too open-ended and there is no clear list of things to be deprecated. Closing.
